azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/internal/BaseAzureComputeApiLiveTest.java

```diff
@@ -114,6 +142,17 @@ private void deleteResourceGroup(String name) {
       api.getResourceGroupApi().delete(name);
    }
 
+   protected ApiContext<AzureComputeApi> getContext() {
+      if (context == null) {
+         Properties properties = new Properties();
+         properties.put("oauth.endpoint", "https://login.microsoftonline.com/tenant-id/oauth2/token");
+         context = ContextBuilder.newBuilder("azurecompute-arm")
+                 .credentials(getIdentity(), getCredential())
+                 .overrides(properties)
+                 .build();
+      }
+      return context;
+   }
```

Comment:
Doing this would mean callers will be creating *another* context, just to access the injector. Instead, you'd better override the [BaseApiLiveTest#create](https://github.com/jclouds/jclouds/blob/master/core/src/test/java/org/jclouds/apis/BaseApiLiveTest.java#L141-L145) method, to create the context and assign it before returning the api.